### PR TITLE
added more tensorboard logs in rl experiments ...

### PR DESCRIPTION
**Pull request by norci**
1. added more tensorboard logs in rl experiments.
1. added Loss values for DDPG policy
1. adjusted stop conditions, for faster testing.
1. refactor.

### CODE REVIEW

File: src/experiments/rl_envs.jl [hunk 1]
```diff
@@ -553,10 +540,33 @@ function RLCore.Experiment(
...
+    hook = ComposedHook(
+        total_reward_per_episode,
+        time_per_step,
+        DoEveryNStep() do t, agent, env
+            with_logger(lg) do
+                @info "training" actor_loss = agent.policy.learner.actor_loss
+                @info "training" critic_loss = agent.policy.learner.critic_loss
+                @info "training" entropy_loss = agent.policy.learner.entropy_loss
+                @info "training" loss = agent.policy.learner.loss
+                @info "training" reward = total_reward_per_episode.reward[end]
```
Comment (findmyway) [inline src/experiments/rl_envs.jl]: Either merge them into a single `@info` expression or add ` log_step_increment = 0` at the end of `@info` except the first one. See more details here: https://philipvinc.github.io/TensorBoardLogger.jl/dev/#Basic-Usage

> Every TBLogger has an internal counter to store the current step, which is initially set to 1. All the data logged with the same @log call will be logged with the same step, and then it will increment the internal counter by 1.

> If you want to increase the counter by a different amount, or prevent it from increasing, you can log the additional message log_step_increment=N. The default behaviour corresponds to N=1. If you set N=0 the internal counter will not be modified. The defaul behaviour for logger lg can be changed executing set_step_increment!(lg, N).

File: src/experiments/rl_envs.jl [hunk 2]
```diff
@@ -614,12 +630,38 @@ function RLCore.Experiment(
...
```
Comment (findmyway) [inline src/experiments/rl_envs.jl]: Same as above

File: src/experiments/rl_envs.jl [hunk 3]
```diff
@@ -689,20 +738,47 @@ function RLCore.Experiment(
...
+            with_logger(lg) do
+                @info "training" actor_loss = agent.policy.actor_loss
+                @info "training" critic_loss = agent.policy.critic_loss
+            end
+        end,
+        DoEveryNEpisode() do t, agent, env
+            with_logger(lg) do
+                @info "training" reward = total_reward_per_episode.rewards[end]
+            end
+        end,
```
Comment (findmyway) [inline src/experiments/rl_envs.jl]: Same as above

**Review by findmyway — CHANGES_REQUESTED** [review]
Great thanks!

I never made the time to do this kind of refactoring. 😂

File: src/experiments/rl_envs.jl (hunk 1 above)
Comment (norci) [reply src/experiments/rl_envs.jl]: Thanks for your help.
I was wondering why the step counts were increased.
